xai_components/xai_connectivity/siibra_connectivity.py

```diff
@@ -5,30 +5,28 @@
 # (c) 2022-2023, TVB Widgets Team
```

Comment:
Should we also update this text with 2024?

Reply:
yep. that would be an update on many files, but we should do it